jbpm-wb-common/jbpm-wb-data-filters-client/src/test/java/org/jbpm/workbench/df/client/filter/FilterSettingsTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.df.client.filter;
+
+import org.dashbuilder.dataset.DataSetLookup;
+import org.dashbuilder.dataset.filter.ColumnFilter;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.dashbuilder.dataset.filter.FilterFactory.equalsTo;
+import static org.junit.Assert.*;
+
+public class FilterSettingsTest {
+
+    private FilterSettings settings;
+
+    @Before
+    public void setup() {
+        settings = new FilterSettings();
+        settings.setDataSetLookup(new DataSetLookup());
+    }
+
+    @Test
+    public void testAddColumnFilter() {
+        final ColumnFilter filter = equalsTo("columnid",
+                                             "value");
+        settings.addColumnFilter(filter);
+
+        assertEquals(1,
+                     settings.getDataSetLookup().getFirstFilterOp().getColumnFilterList().size());
+        assertTrue(settings.getDataSetLookup().getFirstFilterOp().getColumnFilterList().contains(filter));
+    }
+
+    @Test
+    public void testRemoveColumnFilter() {
+        final ColumnFilter filter = equalsTo("columnid",
+                                             "value");
+
+        settings.addColumnFilter(filter);
+        settings.removeColumnFilter(filter);
+
+        assertEquals(0,
+                     settings.getDataSetLookup().getFirstFilterOp().getColumnFilterList().size());
+        assertFalse(settings.getDataSetLookup().getFirstFilterOp().getColumnFilterList().contains(filter));
+    }
+
+}
```

Comment:
I'd merge these two tests into one (_testAddRemoveColumnFilter_).
For example, add two filters, verify both are are there, remove one, verify only one is left.

The reason I'm proposing this is that the second test (_testRemoveColumnFilter_) can be false positive, if adding a filter fails.